music/app/controllers/services_controller.rb

```diff
@@ -11,7 +11,9 @@ def new
   def create
     @resource = OpenStruct.new(date: Date.parse(params[:resource][:date]), body: params[:resource][:body])
```

Comment:
I've added some tests that don't quite test what you asked for, but they get part of the way...